openssl/LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2011 Google Inc.
+Copyright 2011-2017 Google Inc.
```

Comment:
Why?

Reply:
As a Google employee, Google claims copyright over my patches [1]. However, it is not explicitly said whether the copyright year matters or not. I can ask on Monday if you want.

[1] https://opensource.google.com/docs/patching/

Reply:
Ah cool, that makes sense :)